apps/web/.env.example

```
@@ -1,3 +1,2 @@
-VITE_CONVEX_URL=
-VITE_CLERK_PUBLISHABLE_KEY=
-CLERK_SECRET_KEY=
\ No newline at end of file
+BETTER_AUTH_SECRET=
+BETTER_AUTH_URL=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)